web-api/src/storage/postgres.rs

```diff
@@ -315,7 +315,7 @@ impl Database {
                         .push_bind(update.point.to_vec())
                         .push_bind(update.stats.view_count as i32)
                         .push_bind(update.stats.view_time.as_millis() as i64)
-                        .push_bind(now);
+                        .push_bind(time.to_rfc3339());
```

Comment:
:wrench: That should not be needed, `sqlx` has a chrono feature.

```suggestion
                        .push_bind(time);
```